src/components/ListDetail/ListDetail.module.css

```diff
@@ -140,20 +161,19 @@
   }
 }
 
+/**
+ * Before pseudoelement bullet (for non-number variant items)
+ * 1) Set to 9px to center the border
+ * 2) This is the same color used for the vertical line when list style is "ordered"
+ * 3) Using a percentage here instead of --eds-theme-border-radius because this is a cosmetic element (a cicular bullet), not a typical bordered object
+ */
 .list-detail__link-left:not(.list-detail__link-hidden) {
-  /**
-   * Before pseudoelement bullet (for non-number variant items)
-   * 1) Set to 9px to center the border
-   * 2) This is the same color used for the vertical line when list style is "ordered"
-   * 3) Using a percentage here instead of --eds-theme-border-radius because this is a cosmetic element (a cicular bullet), not a typical bordered object
-   */
   &:before {
     content: '';
-    flex-shrink: 0;
```

Comment:
flex-shrink wasn't doing anything here; adding flex-grow: 1 to the first two items in a row (pseudo-bullet, title), then flex-grow: 3 to the arrow on < lg viewports. this keeps the first two items justified left while letting the 3rd item (the arrow) justify right